azure-pipelines.yml

```diff
@@ -697,17 +697,6 @@ stages:
           chmod a+x ${TESTSFOLDER}/test.sh
           ${TESTSFOLDER}/test.sh ${OSNAME} Automation Test
         displayName: Run Automation Tests
-      - task: CopyFiles@2
-        displayName: 'Copy Screenshot to: $(Build.ArtifactStagingDirectory)'
-        inputs:
-          SourceFolder: '$(Build.SourcesDirectory)'
-          Contents: |
-            **/*_test_screenshot.png
-          TargetFolder: '$(Build.ArtifactStagingDirectory)/screenshots'
-      - publish: $(Build.ArtifactStagingDirectory)/screenshots
-        artifact: '$(osName)AutomationScreenshots'
-        condition: and(succeeded(), eq(variables['System.JobAttempt'], '1'))
-        displayName: Publish Screenshot Bundle
```

Comment:
I don't think this block should be removed?

Reply:
commit in; tests running so hopefully back in right